tests/foreman/destructive/test_katello_agent.py

```diff
@@ -153,8 +154,7 @@ def test_positive_upgrade_warning(sat_with_katello_agent):
 
     """
     sat = sat_with_katello_agent
-    ver = sat.version.split('.')
-    target_ver = f'{ver[0]}.{int(ver[1]) + 1}'
+    target_ver = '6.15'
```

Comment:
@vsedmik Do we need to keep this hardcoded? I don't understand the reason for removing the original code related to `target_ver`. Or we can use `SATELLITE_VERSION` that would be better in my opinion than having another constant here. wdyt?
```
from robottelo.constants import SATELLITE_VERSION
target_ver  = float(SATELLITE_VERSION) + 1
```

Reply:
@jameerpathan111 The idea behind is that this test is "6.14.z-only" so the next version is always 6.15. But I don't mind to bring the original code back if you think it's better.

Reply:
Yeah, but it's always good to keep things consistent across the branches, it avoids complications going forward like in the case of cherry picks. Anyway, you have my ACK:)

Reply:
This test cannot pass in branches other than `6.14.z` so I wouldn't be afraid of CPs. Anyway, updated for consistency.